modules/burials/spec/fixtures/pdf_fill/21P-530EZ/overflow.json

```diff
@@ -65,11 +65,11 @@
   "burialCost": 9999999999999999999,
   "burialAllowance": false,
   "plotAllowance": false,
-  "amountGovtContribution": 9999999999999999999,
+  "amountGovtContribution": " 9999999999999999999",
```

Comment:
Is that extra space at the beginning of the string on purpose?

Reply:
@rmtolmach yea it does a rshift. If you look at the kitchen_sink or simple you'll see it